configure.php

```diff
@@ -93,12 +93,24 @@ function remove_composer_script($scriptName) {
     file_put_contents(__DIR__.'/composer.json', json_encode($data, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE));
 }
 
+function remove_readme_paragraphs(string $file): void {
+    $contents = file_get_contents($file);
+
+    file_put_contents(
+        $file,
+        preg_replace('/<!--delete-->.*<!--\/delete-->/s', '', $contents) ?: $contents
+    );
+}
+
 function safeUnlink(string $filename) {
     if (file_exists($filename) && is_file($filename)) {
         unlink($filename);
     }
 }
 
+function fixOsSep(string $path): string {
```

Comment:
rename this to `determineSeparator`

Reply:
Done, I was just trying to make the name shorter